src/Blockcore.Indexer/Storage/Mongo/MongoData.cs

```diff
@@ -785,5 +815,62 @@ private IEnumerable<MapTransactionAddress> PoolToMapTransactionAddress(IEnumerab
             }
          }
       }
+      public void addBalanceRichlist(MapTransactionAddress transaction)         
```

Comment:
Method names should follow .NET Naming guidelines: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-guidelines

Please rename this to "AddBalanceRichlist"